app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -85,7 +85,7 @@ class GithubWebhook extends RequestHandler<Body> {
           }
           break;
         case 'create':
-          branchService ??= BranchService(datastore, rawRequest: stringRequest);
```

Comment:
After many thinkings and testings, my deduction is that:
Cocoon might not be creating a new webhook instance every time we send a request. If the webhook instance is reused over time, then the string request carried by the new event will not make it into the old instance.